src/views/Proposals.vue

```diff
@@ -21,7 +21,7 @@
           <UiButton>{{ $t('proposals.new') }}</UiButton>
         </router-link>
         <router-link
-          v-if="isMember && isEns"
+          v-if="(isAdmin && isEns) || (isOwner && isEns)"
```

Comment:
all admins will have ens names?

Reply:
No, but here the "isEns" is about the space, not about the admin

Reply:
I think this is checking the domain of the space, not the if admin has ENS.

Reply:
```
 key() {
      return this.domain || this.$route.params.key;
    },```